plenum/server/view_change/view_changer.py

```diff
@@ -210,6 +211,11 @@ def is_behind_for_view(self) -> bool:
 
     # EXTERNAL EVENTS
 
+    def on_backup_degradation(self, degraded_backups):
```

Comment:
Maybe we can put all logic related to backup instance removing to a separate class (BackupInstanceFaultyProcessor)?